src/assets/javascripts/govuk-frontend-details.mjs

```diff
@@ -1,30 +1,11 @@
-// JS Module used to combine all the JS modules used in the application into a single entry point,
-//
-// When processed by a bundler, this is turned into a Immediately Invoked Function Expression (IIFE)
-// The IIFE format allows it to run in browsers that don't support JS Modules.
-//
-// Exported items will be added to the window.GOVUK namespace.
-// For example, `export { initAll }` will assign `initAll` to `window.GOVUK.initAll`
-import { nodeListForEach } from 'govuk-frontend/govuk/common'
```

Comment:
Minor: how come we're not using `nodeListForEach` anymore? The new code is longer, so I'm not sure what we gain from the change.

Reply:
Looking at [the nodeListForEach code](https://github.com/alphagov/govuk-frontend/blob/4b5ed0ada3b64722ae2074dfa0db5d4d3a45cb79/src/govuk/common.js#L6), it adds the following to the native looping I use:
1. use of `nodeList.forEach` if it exists
2. uses `.call` to bind the `this` in your callback function to the current node in the list

`nodeList.forEach` is probably a bit faster but we don't use `this` so the benefits seemed a bit minor for the extra code (`nodeListForEach` is actually bit longer than my loop) and addition of a dependency. This is total opinion though and interested to know if you think my version is the right option here.

Reply:
With some hindsight, I think this change adds noise to the history and adds only marginal gains so I've reverted back to `nodeListForEach` in https://github.com/alphagov/notifications-govuk-alerts/pull/99/commits/6fe6012517ce915a4e714571acf7dc86aa103ccc which, I think resolves this (do say if you don't agree).

Reply:
That's OK. I don't mind much either way, and was just surprised / interested by the difference. Since the file is doing stuff with govuk-frontend, I lean towards using their tools, so we're more consistent.

Thanks for changing it back - I was just slow to respond, so feel free to merge with/out that commit.